### ISSUE TITLE
Dilbit volume or bitumen volume calculated incorrectly

### ISSUE THREAD

**Issue by qlangfitt** [arbrandt/OPGEE#250]
The bitumen from mine to dilution (stream 6) calculated on the Heavy Oil Dilution sheet (E12) assumes that the oil_prod input refers to the volume of dilbit. This is apparent because it adjusts the mass down by 1-Fraction_diluent. On the other hand, the same sheet assumes that the oil_prod input refers to only the volume of bitumen when calculating the expected volume of diluted bitumen or heavy oil (M73). One of these is wrong.

OPGEE 3.0 varies in the treatment of oil production volume for Heavy Oil Upgrading, treating it as the bitumen if the upgrader is non-integrated and the SCO if the upgrader is integrated. I think I get the logic behind that, but it's still a bit confusing and doesn't seem to be consistent with OPGEE 2.0.

Looking at OPGEE 2.0 does not clear it up for me. In 2.0, the Active Field sheet states that the Oil Production volume is "before upgrading or dilution", but it seems that on the surface processing sheet it is treated as _after_ dilution. It does treat the volume as the input to upgrading.

This has repercussions on the amount of diluent imported and therefore the upstream impacts of that diluent (and perhaps other impacts).

**Comment by arbrandt** [comment]
I agree that we should take a consistent approach. The two approaches are to have all project types measured volumes either before or after dilution/upgrading. It appears that the Alberta government project databases (https://open.alberta.ca/opendata/alberta-oil-sands-royalty-data1) have projects presenting "cleaned bitumen volume at revenue collection point". This seems like a good endorsement for using that for measuring. It also feels more consistent with crude oil volume calculation. Will implement after checking with jeff on free model status.

**Comment by arbrandt** [comment]
OK, after some work, this is now fixed and consistent. For all possibilities that I had implemented as test fields, we now have the oil production rate (Oil_prod) from the inputs sheet being the bbl/d of raw unprocessed/undiluted/non-upgraded bitumen. For the cases, the 1500 bbl/d default rate appears to correspond to the raw bitumen production. 

For diluted heavy oil and bitumen, we have:
Field 83 (cold lake) is in situ well-based production of bitumen. 1500 bbl/d can be found in computing flow 7, oil flow out of separator
Field 88 (CNRL horizon) is mining based production of bitumen.  We can see 1500 bbl/d can be found in heavy oil dilution M72.

For upgraded heavy oil and bitumen, we have:
Field 85 (Suncor) is integrated mining and upgrading. We see 1500 bbl on upgrading sheet B28
Field 87 (Jackpine) is non-integrated mining and upgrading. We see 1500 bbl on upgrading sheet B28
